doc/whatsnew/2.14.rst

```diff
@@ -383,6 +383,9 @@ Deprecations
 
   Ref #5392
 
+* ``pyreverse``: Resolving and displaying implemented interfaces that are defined by the ``__implements__``
+  attribute has been deprecated and will be removed in 3.0.
+
```

Comment:
Could you put it alongside the changelog for #6712, please ? There is going to be a conflict in the Changelog file in #6688 during rebase (as we burst the changelog file), it would be nice to regroup new changelog entry together until this MR is merged 😄

Reply:
Done 😊